src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -214,7 +218,11 @@ void allocationExport(LocalDate from, LocalDate to, OutputStreamWriter outputWri
                     ALLOCATION_EVENTS, caseTypeCode);
             data.forEach((audit) -> {
                 try {
-                    printer.printRecord(parseAllocationAuditPayload(audit));
+                    String[] parsedAudit = parseAllocationAuditPayload(audit);
+                    if (convert){
+                        parsedAudit = exportDataConverter.convertData(parsedAudit);
```

Comment:
We know the index of caseUuid column here (3) . We could have created a separate method inside exportDataConverter dedicated for caseUUID conversion. The method could take in the index of target column. This would mean we will never try to match UUID that is not caseUUID.

Reply:
This is planned for the 2nd stage of work on this capability.